### PR TITLE
feat: Create the new container Category Table - EXO-61139 - Meeds-io/MIPs#37

### PR DESCRIPTION
**Pull request by sofyenne**
we will implement new  container category with the name `Table` and  on item option `Simple Table Container`  .

### CODE REVIEW

File: web/eXoResources/src/main/webapp/skin/DefaultSkin/portal/webui/component/customization/UIContainerList/Stylesheet.css [hunk 1]
```diff
@@ -1,22 +1,22 @@
...
- * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
- * Lesser General Public License for more details.
- * 
- * You should have received a copy of the GNU Lesser General Public
- * License along with this software; if not, write to the Free
- * Software Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA
- * 02110-1301 USA, or see the FSF site: http://www.fsf.org.
- */
-
+/**
```
Comment (azayati) [inline web/eXoResources/src/main/webapp/skin/DefaultSkin/portal/webui/component/customization/UIContainerList/Stylesheet.css]: keep the same indentation

**Review by azayati — CHANGES_REQUESTED** [review]

File: web/eXoResources/src/main/webapp/skin/DefaultSkin/portal/webui/component/customization/UIContainerList/Stylesheet.css (hunk 1 above)
Comment (sofyenne) [reply web/eXoResources/src/main/webapp/skin/DefaultSkin/portal/webui/component/customization/UIContainerList/Stylesheet.css]: done !

Comment (sofyenne) [reply web/eXoResources/src/main/webapp/skin/DefaultSkin/portal/webui/component/customization/UIContainerList/Stylesheet.css]: done !

File: web/eXoResources/src/main/webapp/skin/DefaultSkin/portal/webui/component/customization/UIContainerList/Stylesheet.css [hunk 2]
```diff
@@ -33,6 +33,11 @@
 .UIContainerList .OneRowContainerLayout	{
 	background: transparent url(background/OneRowContainerLayout.gif) no-repeat left 10px; /* orientation=lt */
 	background: transparent url(background/OneRowContainerLayout-rt.gif) no-repeat right 10px; /* orientation=rt */
+}
+
+.UIContainerList .SimpleTableContainerLayout	{
```
Comment (azayati) [inline web/eXoResources/src/main/webapp/skin/DefaultSkin/portal/webui/component/customization/UIContainerList/Stylesheet.css]:
```suggestion
.UIContainerList .SimpleTableContainerLayout {
```

File: web/eXoResources/src/main/webapp/skin/DefaultSkin/portal/webui/component/customization/UIContainerList/Stylesheet.css [hunk 3]
```diff
@@ -33,6 +33,11 @@
...
+	background: transparent url(background/SimpleTableContainerLayout.gif) no-repeat left 10px; /* orientation=lt */
+	background: transparent url(background/OneRowContainerLayout-rt.gif) no-repeat right 10px; /* orientation=rt */
```
Comment (azayati) [inline web/eXoResources/src/main/webapp/skin/DefaultSkin/portal/webui/component/customization/UIContainerList/Stylesheet.css]:
```suggestion
	background: transparent url(background/SimpleTableContainerLayou-rt.gif) no-repeat right 10px; /* orientation=rt */
```

File: web/portal/src/main/webapp/groovy/portal/webui/container/UISimpleTableContainer.gtmpl [hunk 4]
```diff
@@ -0,0 +1,98 @@
...
+	if(componentId.startsWith("UIContainer-")){
+		uicomponent.setId(componentId.substring("UIContainer-".length()));
+	}
+  
+  if (uiPortalApp.isEditing())
+  {
+     def reqJS = rcontext.getJavascriptManager().require("SHARED/portal", "portal");
+     reqJS.addScripts("portal.UIPortal.initMouseHover('" + uicomponent.id + "');");
+%>  
+<div class="UIContainer Simple-Table-Container<%=uiComponentClass%> EdittingContainer<%=uicomponent.getPermissionClasses()%>"
```
Comment (azayati) [inline web/portal/src/main/webapp/groovy/portal/webui/container/UISimpleTableContainer.gtmpl]:
```suggestion
<div class="UIContainer simpleTableContainer <%=uiComponentClass%> EdittingContainer<%=uicomponent.getPermissionClasses()%>"
```

File: web/portal/src/main/webapp/groovy/portal/webui/container/UISimpleTableContainer.gtmpl [hunk 5]
```diff
@@ -0,0 +1,98 @@
...
+				id="${uicomponent.id}" ${cssStyle}>
+				<%=uiComponentClass%>
+<%
+  }
+  else
+  {   
+%>
+<div class="UIContainer Simple-Table-Container<%=uiComponentClass%> <%=hasAccessPermission?"": "ProtectedContainer"%>" id="${uicomponent.id}" ${normalCssStyle}>
```
Comment (azayati) [inline web/portal/src/main/webapp/groovy/portal/webui/container/UISimpleTableContainer.gtmpl]:
```suggestion
<div class="UIContainer simpleTableContainer <%=uiComponentClass%> <%=hasAccessPermission?"": "ProtectedContainer"%>" id="${uicomponent.id}" ${normalCssStyle}>
```

File: web/portal/src/main/webapp/groovy/portal/webui/container/UISimpleTableContainer.gtmpl [hunk 6]
```diff
@@ -0,0 +1,98 @@
...
+
+	String cssStyle = "style=\"";
+	String uiComponentWidth = uicomponent.getWidth();
+	String uiComponentHeight = uicomponent.getHeight();
+	if(uiComponentWidth != null) cssStyle += "width: "+uiComponentWidth+";"
+	if(uiComponentHeight != null) cssStyle += "height: "+uiComponentHeight+";"
+	cssStyle += "\"";
+	String normalCssStyle = cssStyle.substring(0,cssStyle.length()-1) ;
+	normalCssStyle+= "margin-top: 20px;"
+        normalCssStyle += "\"";
```
Comment (azayati) [inline web/portal/src/main/webapp/groovy/portal/webui/container/UISimpleTableContainer.gtmpl]:
```suggestion
	String cssStyle = "";
	String uiComponentWidth = uicomponent.getWidth();
	String uiComponentHeight = uicomponent.getHeight();
	if(uiComponentWidth != null || uiComponentHeight != null) cssStyle = "style=\"";
	if(uiComponentWidth != null) cssStyle += "width: "+uiComponentWidth+";"
	if(uiComponentHeight != null) cssStyle += "height: "+uiComponentHeight+";"
	if(cssStyle.length() > 0) cssStyle += "\"";
```

**Review by azayati — CHANGES_REQUESTED** [review]

**Review by azayati — APPROVED** [review]
LGTM

**Pull request merged**